### PR TITLE
Update Gizra Way text

### PR DESCRIPTION
**Pull request by IshaDakota**
#459 @RachelBaram Here's a start.

### CODE REVIEW

**Comment by RachelBaram** [comment]
@IshaDakota Thank you for opening this PR for me. This should come out to 6 rows in each section. How does it look? 

Does it matter if the budget goggles link is absolute and not relative? Just trying to learn best practices.

**Comment by IshaDakota** [comment]
fireshot capture 83 - design web development i gizra - http___0 0 0 0_9000_ about

**Comment by IshaDakota** [comment]
> Does it matter if the budget goggles link is absolute and not relative?

Internal links should be relative, and not targeted (unless a new tab is desired <--- use with extreme caution):

`<a href=/content/budget-goggles/">budget goggles</a>`

**Comment by IshaDakota** [comment]
fireshot capture 84 - google hangouts_ - https___hangouts google com_hangou

File: app/_includes/about.html [hunk 1]
```diff
@@ -71,20 +73,25 @@ <h3 class="list-group-item-heading">Effective Communication</h3>
             <div class="list-group">
               <div class="list-group-item">
                 <h3 class="list-group-item-heading">Meticulous Execution</h3>
-                <p class="list-group-item-text text-small">A complete deployment and testing system is built 
-                  into our projects from day one, growing as the code does. We use
-            <a href="http://shoov.io/" target="_blank">Shoov</a>, a Gizra-developed testing toolkit,
-            which ensures design is optimal from visual, interaction, and technical standpoints.</p>
+                <p class="list-group-item-text text-small">
+                  A complete deployment and testing system is built into our projects
+                  from day one, growing as the code does. We use <a href="http://shoov.io/" target="_blank">Shoov</a>, a Gizra developed
```
Comment (IshaDakota) [inline app/_includes/about.html]: let's not use targets even on external links. The assumption that the user has a back button and knows how to use it is generally a good one ;)

**Pull request merged**